lib/Component.php

```diff
@@ -67,7 +58,7 @@ public function getScripts() {
         continue;
       }
 
-      $scripts[] = $asset_url . $script;
+      $scripts[] = $this->base_url . $script;
```

Comment:
This assumes that $this->base_url always ends with a slash (or that $script begins with one). It is probably a good idea to build the path using some function/method instead. Some examples: https://stackoverflow.com/questions/1091107/how-to-join-filesystem-path-strings-in-php